lib/osutil/lowprio_linux.go

```diff
@@ -42,23 +42,33 @@ func ioprioSet(class ioprioClass, value int) error {
 // SetLowPriority lowers the process CPU scheduling priority, and possibly
 // I/O priority depending on the platform and OS.
 func SetLowPriority() error {
+	// Process zero is "self", niceness value 9 is something between 0
+	// (default) and 19 (worst priority).
+	const (
+		pidSelf       = 0
+		wantNiceLevel = 9
+	)
+
+	if cur, _ := syscall.Getpriority(syscall.PRIO_PROCESS, pidSelf); cur >= wantNiceLevel {
```

Comment:
Why are we not handling the error?

Reply:
I'm being lazy and assuming that if there is an error I'm not to get a positive integer back

Reply:
Wouldn't it make sense to abort if the niceness is anything but the default value? Line in if the user sets niceness manually, it's nicer (haha) to not mess with it at all.